static/app/components/organizations/timeRangeSelector/index.tsx

```diff
@@ -448,7 +450,7 @@ class TimeRangeSelector extends PureComponent<Props, State> {
                 maxHeight={400}
                 detached={detached}
                 items={items}
-                searchPlaceholder={t('Filter time range')}
+                searchPlaceholder={t('Provide a time range')}
```

Comment:
Yeah I was grappling with that a bit - do you have any good ideas for how to show that hint? I was thinking either in the placeholder (but not very much room there) or empty items text

Reply:
Yea it is unfortunate there is not enough space in the placeholder, maybe enough for one example. The empty text is also a good suggestion, would probably defer this to @vuluongj20 in case he has any thoughts.